runner/src/runner/pipeline.clj

```diff
@@ -58,12 +59,20 @@
 
 (defn resourceful-step
   "Create a resource mounted image for a step if it needs it."
-  [db-client step group name image run-id]
+  [{:keys [database producer]}
+   {:keys [group name image run-id]}
```

Comment:
This would fail the tests for resourceful-steps I think?

Reply:
It's not possible currently to make the tests passing with https://github.com/bob-cd/resource-git/issues/3